src/OrchardCore/OrchardCore.Users.Abstractions/UserOptions.cs

```diff
@@ -30,5 +31,11 @@ public string ExternalLoginsUrl
             get => _externalLoginsUrl;
             set => _externalLoginsUrl = value.Trim(' ', '/');
         }
+
+        public string ChangePasswordConfirmationPath
```

Comment:
I renamed **ChangePasswordConfirmation** to **ChangePasswordConfirmationPath** because it represents an action path like the property **LogoffPath**.